cli/cmd/plugin/ui/web/tanzu-ui/src/shared/services/awsDeployment.tsx

```diff
@@ -41,21 +41,14 @@ const useAwsDeployment = () => {
             createCloudFormationStack: false,
             clusterName: awsData[AWS_FIELDS.CLUSTER_NAME],
             controlPlaneFlavor: awsData[AWS_FIELDS.CLUSTER_PLAN],
-            controlPlaneNodeType: nodeType[awsData[AWS_FIELDS.NODE_PROFILE]],
+            controlPlaneNodeType: nodeType[awsData[AWS_FIELDS.NODE_PROFILE]]['nodeType'],
             bastionHostEnabled: awsData[AWS_FIELDS.ENABLE_BASTION_HOST],
             machineHealthCheckEnabled: awsData[AWS_FIELDS.ENABLE_MACHINE_HEALTH_CHECK],
             vpc: {
                 cidr: awsData[AWS_FIELDS.VPC_CIDR],
                 vpcID: '',
                 // TODO: single subregion name populated from region selection; but does not support multi-az/HA
-                azs: [
-                    {
-                        name: awsData[AWS_FIELDS.REGION] + 'a',
-                        workerNodeType: nodeType[awsData[AWS_FIELDS.NODE_PROFILE]],
-                        publicSubnetID: '',
-                        privateSubnetID: '',
-                    },
-                ],
+                azs: nodeType[awsData[AWS_FIELDS.NODE_PROFILE]]['vpc'],
```

Comment:
This feels like there is a bit of a naming mismatch or the 'vpc' data that we are pulling into 'azs' here is not actually a full 'vpc' object. This might be easier to walk through over zoom when you are back online.

Reply:
Already corrected this